eval.go

```diff
@@ -588,6 +584,7 @@ func (d *indexData) gatherBranches(docID uint32, mt matchTree, known map[matchTr
 
 	var branches []string
 	id := uint32(1)
+	repoIdx := d.repos[docID]
 	for mask != 0 {
 		if mask&0x1 != 0 {
 			branches = append(branches, d.branchNames[repoIdx][uint(id)])
```

Comment:
minor refactor suggestion
```suggestion
	branchNames := d.branchNames[d.repos[docID]]
	for mask != 0 {
		if mask&0x1 != 0 {
			branches = append(branches, branchNames[uint(id)])
```